sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -74,12 +75,13 @@ export interface Sender {
    * Sends a batch of messages to the associated service-bus entity.
    *
    * @param {ServiceBusMessageBatch} messageBatch A batch of messages that you can create using the {@link createBatch} method.
+   * @param options - Options for aborting and tracing.
```

Comment:
In general, I wonder if it would be better to say `cancellation` instead of `aborting`.

Reply:
You are right (at least, in what we've used before):

`A signal used to cancel the send operation.`

Reply:
Swapped to @ramya-rao-a 's wording:

   * @param options - Options bag to pass an abort signal or tracing options.